browser-test/src/applicant/application_version_fast_forward.test.ts

```diff
@@ -0,0 +1,1000 @@
+import {test, expect} from '../support/civiform_fixtures'
+import {
+  AdminQuestions,
+  AdminPrograms,
+  ApplicantQuestions,
+  loginAsAdmin,
+  loginAsProgramAdmin,
+  loginAsTestUser,
+  logout,
+  closeWarningMessage,
+  AdminPredicates,
+} from '../support'
+import {QuestionSpec} from '../support/admin_programs'
+import {Browser, Locator, Page} from '@playwright/test'
+
+test.describe('Application Version Fast Forward', () => {
+  test.beforeEach(async ({request}) => {
+    await test.step('Clear database', async () => {
+      await request.post('/dev/seed/clear')
+    })
+  })
+
+  test('all major steps', async ({browser}) => {
+    const programName = 'program-fastforward-example'
+
+    const civiformAdminActor = await FastForwardCiviformAdminActor.create(
+      programName,
+      browser,
+    )
+    const applicantActor = await FastForwardApplicantActor.create(
+      programName,
+      browser,
+    )
+    const programAdminActor = await FastForwardProgramAdminActor.create(
+      programName,
+      browser,
+    )
+
+    /*
+
+      Program definitions
+
+      Program: v1       Add (A, B, C, D, E)
+        Block: First
+          Question: A
+        Block: Second
+          Question: B  ---> Eligibility
+        Block: Third
+          Question: C  ---> Eligibility
+        Block: Fourth
+          Question: D
+        Block: Fifth
+          Question: E
+
+      Program: v2       Remove (C, D), move (A, E), add (F, G), no-change (B)
+        Block: First
+          Question: G
+        Block: Second
+          Question: B  ---> Eligibility
+        Block: Third
+          Question: A
+        Block: Fourth
+          Question: E
+        Block: Fifth
+          Question: F
+
+      Program: v3       Add (H), no-change (G, B, A, E, F)
+        Block: First
+          Question: G
+        Block: Second
+          Question: B  ---> Eligibility
+        Block: Third
+          Question: A
+        Block: Fourth
+          Question: E 
+                    H ---> Eligibility
+        Block: Fifth
+          Question: F
+    */
+
+    /*
+      High level overview
+
+      - Civiform Admin creates program v1
+      - Applicant fills out application to program v1; does not submit
+      - Civiform Admin creates program v2
+      - Applicant goes into the edit appliction page and stays in there
+      - Civiform Admin creates program v3
+      - Applicant, still on the edit page, submits application for program v1
+      - Program Admin can view applications as expected for program v1
```

Comment:
iiuc this is _not_ fast forwarding. Should the test name or this comment clarify that?